maven/WFLY-14934_Server_Plugin.adoc

```diff
@@ -0,0 +1,217 @@
+= WildFly Maven Plugin to produce fully configured WildFly server
+:author:           Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Introduce new Maven tooling to provision a WildFly installation, configure the server and deploy application into it. 
+The Maven tooling configuration points are similar to what WildFly Bootable JAR Maven plugin offers.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14934[WFLY-14934]
+
+=== Related Issues
+
+* Deprecation of some existing wildfly-maven-plugin features: https://issues.redhat.com/browse/WFMP-130[WFMP-130]
```

Comment:
@jfdenise could you please add https://issues.redhat.com/browse/WFMP-138 to the list of related issues?

Reply:
Thanks, added.